tox.ini

```diff
@@ -26,7 +26,7 @@
 
 [tox]
 envlist =
-  py37,py38,py39
+  py38,py39
```

Comment:
Should `py310` be added here?

Reply:
Sure. Though I still feel the purpose of this file should be cleared up, too, as without it being plugged into the CI system there isn't really guarantee that it keeps working (maybe it's just me who gets frustrated/confused when there are multiple flavours of unused environment files in a project at different stages of maintenance)